### DISCUSSION TITLE
A Brief Rulei A&Q

### DISCUSSION THREAD

**Discussion by Saugia** [endless-sky/endless-sky#10893] [Lore]
## Introductory Note

MZ once created a lore document on Christmas explaining his thoughts on how the game's lore played out. While not all of it is canon, a majority of it was both insightful and interesting at the time and even now supports a good portion of the game's lore.

Happy New Years! This little "A & Q", as I'd like to call it, is a similarly festively-timed exposition for the Rulei. Rather than write a whole discussion, I will be giving some snippets of Rulei lore that will answer some existing speculation, and add more questions, hence it is "Answers then Questions". This can still be tentative, as there might be future factions or lore that will add to the Rulei's background to the game in some manner.

If you wish to keep the Rulei a mystery whispering in the back of your head, please stop here. If you wish to gain a bit more knowledge of them, continue at your own risk! As more content for the Rulei are added to the game, either I will update this or I will make a full Rulei lore post, similar to the Ka'het one on the wiki.

---

## The Rulers of the East

Who were the Rulei?
- The Rulei, Ruleians, or self-proclaimed "Precursors", were the fourth spacefaring species in the galaxy. Just before them were the Quarg, the Drak, and then the so-called "First-born". During a majority of their time in the galaxy, the Rulei knew only of the Quarg, and not the Drak.

How capable were the Rulei?
- The Rulei were very technologically capable, learning at an exponential rate and rivaling that of the Quarg, also being Tier 3 in many respects, though the Quarg far surpass the Rulei in the growth rate and scientific understanding of their own. The Rulei were similarly capable of making massive megastructures, such as the one in G-9139, with the height of Ruleian technology peaking at this scale of construction and manipulation of stellar masses. These megastructures were typically made in the the Rulei's own image in some shape or form.

- Like the Quarg and as evident by G-9139, the Rulei were versed in some sort of graviton-based tech, or something akin to it.

Is the Quarg Ring in Aleit for the Rulei?
- Giaru Gegno is the Quarg's ring for the Gegno, however it was indeed once the ring for the Rulei. This is further shown as the Quarg's nearby "tourist station" is named Ruelogakk. The larger bays in the stations are not, however, due to the Rulei, but simply that Gegno ships are consistently larger than human ships or others.

What is the Solemnity?
- The Solemnity is not so much a ship as it is an organic lifeform, made of a sort of living crystalline-like bio-substance, composed of thousands of Ruleian minds merged together. Similar to Archons, the Solemnity can speak telepathically. It is more like a collective than it is a hivemind, as there still remains a sense of "individuals", though they generally speak as one.

- In the Solemnity's classification, the sole Rulei ship in the game, the prefix "vyu" in vyu-Ir represents a certain group of Rulei, and the suffix "Ir" is a sort of ship designation.

How are the Rulei able to speak to me?
- The Solemnity reads your mind to be able to speak to you, learning your language through it. This also means the Solemnity can gain the knowledge of whatever you have knowledge of or have seen.

What's going on with the planets in Yranjiu?
- The Ruleian planets in Yranjiu are covered in a sort of "primoridal soup" similar to the nature of the Solemnity but on a much grander scale. It also similarly emits voices, and subconscously pulls at your own consciousness as if wanting yours to be part of theirs. If you touched this organic material, you would be assimilated into it in a matter of moments. This is also true for non-organic materials.

Where is Rulei Territory?
- All of Gegno space, the Yranjiu cluster, and the Umbral Reach were once Rulei space. The Vyrmeid area was not part of Rulei space, but they did explore it.

- While the Rulei eventually controlled a not insignificant region of the map and fell in the same Tier bracket, they wouldn't have ever been able to challenge the Quarg on a galactic level.

Why isn't there more Ruleian remains?
- The lack of an abundance of Ruleian societal ruins and megastructures throughout the rest of their large, former region suggests that much of it had been removed at some point prior to the dwelling of the Gegno.

Is Yranjiu the Rulei home system?
- The Ruleian home system does not appear on the star map.

Happy theorizing!. Don't overdo it or else you won't stop hearing the voices! :)

**Comment by Arachi-Lover** [comment]
Nice!

`The Rulei, Ruleians, or self-proclaimed "Precursors", were the third spacefarring species in the galaxy. Just before them were the Quarg, the Drak, and then the so-called "First-born". During a majority of their time in the galaxy, the Rulei knew only of the Quarg, and not the Drak.`

The fourth you mean then? First-born -> Drak -> Quarg -> Rulei?
~~Or is this like in programming where we start at 0?~~

Other than that, mandatory questions that ***THE PEOPLE***:tm: must know:

Height?

Pop count?
.
.
.
.
.
.
.
.
.
~~Also poem questionmark?~~

**Reply by Saugia** [reply]
Height and pop count are [REDACTED]. Thanks for asking!

(Didn't you make a poem for them already?)
